data-prepper-logstash-configuration/src/main/antlr/Logstash.g4

```diff
@@ -0,0 +1,138 @@
+grammar Logstash;
```

Comment:
Please include Java docs for this grammar file and add `@since 1.2`

Reply:
I think adding a java doc for files other than Java classes is not useful.

Reply:
My bad, I meant adding a comment for what this grammar file consists of.